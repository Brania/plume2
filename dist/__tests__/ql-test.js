"use strict";
const ql_1 = require("../ql");
describe('ql test suite', () => {
    it('ql init', () => {
        const helloQL = ql_1.default('helloQL', [
            'loading',
            (loading) => loading
        ]);
        expect(1).toEqual(helloQL.id());
        expect('helloQL').toEqual(helloQL.name());
        expect(helloQL.lang()).toMatchSnapshot();
    });
});
