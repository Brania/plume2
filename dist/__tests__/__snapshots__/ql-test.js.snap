exports[`ql test suite ql init 1`] = `
Array [
  "loading",
  [Function],
]
`;
